src/sql/workbench/services/contextualization/common/interfaces.ts

```diff
@@ -30,14 +30,9 @@ export interface IServerContextualizationService {
 	getProvider(providerId: string): azdata.contextualization.ServerContextualizationProvider;
 
 	/**
-	 * Generates server context
-	 * @param ownerUri The URI of the connection to generate context for.
+	 * Contextualizes the provided URI for GitHub Copilot.
+	 * @param uri The URI to contextualize for Copilot
+	 * @returns A promise that doesn't need to be awaited since contextualization is done in the background.
```

Comment:
nit: It's up to the caller to decide this. I'd suggest just saying what will have happened when the promise completes